src/Interpreters/DatabaseCatalog.cpp

```diff
@@ -732,6 +794,10 @@ void DatabaseCatalog::dropTableDataTask()
             LOG_INFO(log, "Will try drop {}", table.table_id.getNameForLogs());
             tables_marked_dropped.erase(it);
         }
+        else
+        {
+            LOG_TRACE(log, "Not found any suitable tables to drop, still have {} tables in drop queue", tables_marked_dropped.size());
```

Comment:
This loos too noisy, maybe this can be dropped? Or maybe the interval for the dropTableDataTask should be increased?

Reply:
Yes, it's too annoying. Removal is in our checklist: https://github.com/ClickHouse/ClickHouse/issues/16367

Reply:
You can help remove right now.

Reply:
I guess simply removing it not a great way to go, I have an idea - let's convert this message into metric, thus information will be still available via system table for debugging.

Reply:
Actually there is `TablesToDropQueueSize` already, so looks like message can be simply removed? @tavplubix ?

Reply:
> [In reply to Alexey Milovidov]
Если сильно раздражает — можно просто удалить эту строчку из кода. Исправлю по-нормальному когда доразбираюсь со сломанными тестами.

Reply:
No problem, I can maintain this patch (80656b16910bf11cfcf0ac07e303b42104e7cb27) in my fork until it will be fixed in a normal way (to avoid CPU burn on CI for oneline change)